tuxemon/core/components/player.py

```diff
@@ -453,7 +453,7 @@ def get_collision_dict(self, game):
         npc_positions = set()
 
         # Get all the NPC's tile monsters_in_play so we can check for collisions.
-        for npc in game.npcs:
+        for npc in game.npcs.items():

```

Comment:
Actually, I meant to use Values. The slugs aren't necessary here, since we have to traverse the entire list anyhow. I didn't look deeply into this method, so it might be that using slugs *could* improve it somehow, but as a first pass change I wanted to be minimalist.